src/util/expr.h

```diff
@@ -12,9 +12,10 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #define OPERANDS_IN_GETSUB
 
-#include <functional>
+#include "mp_arith.h"
```

Comment:
Why is this additional header now necessary when none of the changes below depends on it?!

Reply:
indeed, fixed.